test/puppetlabs/trapperkeeper/services/metrics/metrics_service_test.clj

```diff
@@ -61,20 +61,20 @@
            (is (= 200 (:status resp))))))
 
       (testing "register should add a metric to the registry"
-        (let [svc (app/get-service app :MetricsService)
-              registry (metrics-protocol/get-metrics-registry svc "pl.foo.reg")]
-          (metrics/register registry
-                            (metrics/host-metric-name "localhost" "foo")
-                            (metrics/gauge 2))
-          (let [resp (http-client/get
-                      (java.net.URLDecoder/decode
-                       (str "http://localhost:8180/metrics/v1/mbeans/"
-                            "pl.foo.reg:name=puppetlabs.localhost.foo")))
+        (doseq [[domain metric] [["pl.test.reg" "foo"] [:pl.test.reg "bar"]]]
```

Comment:
I think I wrote `don't` when I meant `do` there. I think that it's better to have string domain and keyword domain that are the same when wrapped in `name` be the same metric registry. I think that if that weren't the case that it would be weird  - especially given that all that `pe-trapperkeeper-metrics` does right now is take the keyword domain and use `name` to convert it to a string to use as the domain (same as this does, but without allowing a string domain), and so if we wanted to have string domain and keyword domain yield different registries we would probably have to do something super janky. So I think the current behavior is right, I just was wondering whether it would be worthwhile to assert this behavior in a test? Don't have a strong opinion on that, just wondering.